Assets/Project/Scripts/GamePlayScene/Tile/HolyTileController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Project.Scripts.GamePlayScene.Bottle;
+using Project.Scripts.Utils.Definitions;
+using UnityEngine;
+
+namespace Project.Scripts.GamePlayScene.Tile
+{
+    public class HolyTileController : AbstractTileController
+    {
+        protected override void Awake()
+        {
+            base.Awake();
+            bottleHandler = new HolyTileBottleHandler();
+        }
+
+        public override void Initialize(int tileNum)
+        {
+            base.Initialize(tileNum);
+
+            #if UNITY_EDITOR
+            name = TileName.HOLY_TILE;
+            #endif
+        }
+
+        private sealed class HolyTileBottleHandler : DefaultBottleHandler
+        {
+            public override void OnBottleEnter(GameObject bottle)
+            {
+                // 親ボトルを無敵状態にする
```

Comment:
```suggestion
                if (bottle.GetComponent<AbstractBottleController>() == null) return;
                // 親ボトルを無敵状態にする
```
前回のレビューで言うべきだったけど、ここ一応チェックしといた方がいいかも

Reply:
07b36cf